### ISSUE TITLE
ShellCheck highlights several issues with the file - Shellcheck could be disabled, or issues resolved

### ISSUE THREAD

**Issue by jamie-wearsafe** [rcaloras/bash-preexec#132]
It can be disabled by adding this line to the shell file:

```bash
# shellcheck disable=SC2292,SC2250,SC2086,SC1087,SC2248,SC2001,SC1007,SC2034
```
Issues can be seen by adding the script to the [shellcheck online editor](https://www.shellcheck.net/):
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/52926764/175314677-408eb38d-3901-42dd-b292-3dc6212ba3c9.png">

**Comment by dimo414** [comment]
Feel free to send a PR addressing the ShellCheck findings, but it would be good to limit the disable directives to specific offending lines where possible.

**Comment by rcaloras** [comment]
Should be addressed in #135
